components/channel_layout/message_indentifier_router.jsx

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This file is going to conflict with the changes from #189, which should probably go in first since the community member has already had to rebase it on top of the react-router changes

Reply:
Sounds good. I will wait for it, then merge those changes in.

Reply:
@hmhealey I am going to wait for #189 anyway, so it will be next release I assume. Plus, this is really just code cleanup, so there is no downside to waiting

Reply:
FYI #189 is now merged.

Reply:
And I have merged master into this branch